pyhf/modifiers/normfactor.py

```diff
@@ -0,0 +1,25 @@
+import logging
+log = logging.getLogger(__name__)
+
+from . import modifier
+
+@modifier
+class normfactor(object):
+    is_constrained = False
+
+    def __init__(self, nom_data, modifier_data):
+        self.n_parameters = 1
+        self.suggested_init = [1.0]
+        self.suggested_bounds = [[0, 10]]
+
+    def alphas(self, pars):
+        raise NotImplementedError
+
+    def expected_data(self, pars):
+        raise NotImplementedError
+
+    def pdf(self, a, alpha):
+        raise NotImplementedError
+
```

Comment:
the `alphas`,`expected data` and `pdf` methods only pertain to the constraint term. As these are unconstrained modifiers, they should not only not be implemented but not be methods at all, as they're not part of the interface for unconstarined modifiers

Reply:
ok. will fix.

Reply:
done